<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AQI</class>
 <widget class="QWidget" name="AQI">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>659</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>空气质量详解图</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 206, 233);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>输入股票代码</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="stock_id"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>起始时间</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="start"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>终止时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="end"/>
     </item>
     <item>
      <widget class="QPushButton" name="draw">
       <property name="text">
        <string>画图</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear">
       <property name="text">
        <string>清空</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dayK">
       <property name="text">
        <string>日K线</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="weekK">
       <property name="text">
        <string>周K线</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="monthK">
       <property name="text">
        <string>月K线</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QWebEngineView" name="webEngineView"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../source/apprcc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>clear</sender>
   <signal>clicked()</signal>
   <receiver>stock_id</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>697</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
